server/controller/apis/resourceController.js

```diff
@@ -85,18 +85,30 @@ exports.getAllSharedResourcesForUser = async ( req, res ) => {
  * @param {*} res 
  */
 exports.getAllActiveResourcesForUser = async ( req, res ) => {
+    // get the auth user id from either the basic auth header or the session
+    let authUserId;
+    if( req.user ) {
+        authUserId = req.user.id;
+    }
+    else if( req.session.authUser ) {
+        authUserId = req.session.authUser.id;
+    }
+
     // get all the active resources
-    let resources = await resourceService.getAllActiveResourcesForOwner();
+    let resources = await resourceService.getAllActiveResourcesForOwner(authUserId);
     
     res.set( "x-agora-message-title", "Success" );
     res.set( "x-agora-message-detail", "Returned all resources" );
     res.status( 200 ).json( resources );
 }
 
 exports.getResourceById = async ( req, res ) => {
-    // get all the active resources by user 
-    let resource = await resourceService.getAllActiveResourcesForOwnerById( req.session.authUser.id, req.params.id );
-    if(resource) {
```

Comment:
Discussed the issue you ran into earlier with @Mike-Volchko, but @briangormanly makes a great point. This will need the same authorization check that you added above so we don't unintentionally circumvent visibility. resourceService.js doesn't do any visibility check, so the authorization check needs to occur in the controller.